.travis.yml

```diff
@@ -16,7 +16,9 @@ env:
     - PUPPET_GEM_VERSION="~> 4.1.0"
     - PUPPET_GEM_VERSION="~> 4.2.0"
     - PUPPET_GEM_VERSION="~> 4.3.0"
-    - PUPPET_GEM_VERSION="~> 4"
+    - PUPPET_GEM_VERSION="~> 4.4.0"
+    - PUPPET_GEM_VERSION="~> 4.5.0"
+    - PUPPET_GEM_VERSION="~> 4.6.0"
     - PUPPET_GEM_VERSION="~> 4" STRICT_VARIABLES="yes"
```

Comment:
we can drop STRICT_VARIABLES if we are using puppetlabs_spec_helper > 1.2 as it is now the default

https://github.com/puppetlabs/puppetlabs_spec_helper#initializing-puppet-for-testing